src/cheri/riscv-unpriv-integration.adoc

```diff
@@ -257,6 +257,25 @@ Capabilities with `CT≠0` are sealed against modification and cannot be derefer
 Instructions that operate on capabilities will produce a result with a cleared {ctag} if the source capability is sealed and the operation would alter its address, bounds, or permissions.
 Extensions that augment capability metadata must describe their interaction(s) with sealed capabilities.
 
+Given a capability with `CT=0`,
```

Comment:
```suggestion
#START: New since ARC review#
Given a capability with `CT=0`,
```

Maybe worth adding a start + end to make it obvious which part is new?